src/components/Header/Header.js

```diff
@@ -1,40 +1,42 @@
 import React from "react";
 import logo from "./logo.png";
-import { MdShoppingCart, MdSearch } from "react-icons/md";
+import { MdShoppingCart } from "react-icons/md";
 import { Link } from "react-router-dom";
+import Search from "../Search/Search.component";
+import CartContext from "../../state/CartContext";
 
 function Header() {
+  const { state } = React.useContext(CartContext);
+
+  function totalCartItems() {
```

Comment:
Something like this?

```
const { state } = React.useContext(CartContext);
const [total, setTotal] = React.useState(
    state.cart.reduce((a, b) => a + b.quantity, 0)
);
```

I thought about it, but since I did not use the setTotal I only used a function, I just added it and it will be on the upcoming commit.

Reply:
Actually, i'm using the CartContext now